<launch>
    <node name="command_publisher" pkg="rviz_pose_publisher" type="controller.py" output="screen" clear_params="true">
        <param name="sat_x" value="1" />
        <param name="sat_y" value="1" />
        <param name="sat_rot" value="1" />
    </node>
</launch>
